src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import "./navbar.css";
+import NavbarOptions from "./NavbarOptions";
+import Button from "../Button/Button";
+
+export default function Navbar() {
+  const [toggle, setToggle] = useState(false);
+
+  const toggleHandler = () => {
+    setToggle(!toggle);
+  };
+
+  const hamburgerIcon = <i className="fa fa-bars" onClick={toggleHandler}></i>;
+  const cancelIcon = <i className="fa fa-times " onClick={toggleHandler}></i>;
+
+  const NavMenu = NavbarOptions.map((btn) => (
+    <li className="navbarMenu">
+      <a href={btn.href} className="navbarPropertyLink">
```

Comment:
You may want to use React Link component instead of the anchor tag to avoid refreshing the page when routing to another URL within your application.